packages/react-components/package.json

```diff
@@ -9,6 +9,9 @@
     "va",
     "vads"
   ],
+  "files": [
+    "index.js"
+  ],
```

Comment:
Allows `core` to import the index file easily.

Reply:
Bonus points for also adding a `module` property.

It's an unofficial property, but [Rollup documents it](https://rollupjs.org/guide/en/#publishing-es-modules).